src/Components/Atoms/DropdownOptionAtom/index.tsx

```diff
@@ -0,0 +1,11 @@
+interface AccountOption {
+    accountOption: string
+}
+
+function DropdownOptionAtom ({accountOption}: AccountOption) {
+    return (
```

Comment:
You've de-structured the props like this here but differently in the other atoms. Matt might want it consistent